src/components/ScreeningCard.vue

```diff
@@ -47,6 +62,12 @@ export default {
       return `${hours}h ${minutes} min`;
     },
   },
+  mounted() {
+    this.$root.$on("getDate", (dt) => {
+      this.date = dt;
+      console.log(this.date);
+    });
```

Comment:
It's awesome that you found this "event bus" pattern and you can use it 👏 
However I personally think that this is not very reasonable usage.

I described a little bit here why I don't think it's a good pattern for most cases:
https://github.com/monterail/kbm/blob/master/kbm/guidelines/development/vue.md#patterns-to-avoid

If you absolutely need communication between components other than "props-emits", you can use vuex.

In this particular case maybe it would be nice to rethink the communication because props should be probably enough here 😉

Reply:
Thanks! The truth is a had a huge problem with coding this functionality and I'm not quite sure, where this logic should be, because these two components (DateButton and ScreeningCard) are separated from each other...

Reply:
if you have two separate components A and B and both of them share some state (in your example it's selected date), you have two main options:

1. Move this state to vuex and both comonents should read it from there (and update it calling store actions)
2. Move this state to some parent component C that has both of them as children - then pass this state as a prop to A and B. If a component A want to change this value it simply emits some event and parent component C can react to this.